torch/_torch_docs.py

```diff
@@ -3836,6 +3836,39 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins, *, min=None, max=None, weight=None, density=False) -> (Tensor, Tensor)
+
+Computes the histogram of a tensor. The histogram is computed over the flattened values.
```

Comment:
Could these sentences be combined, and clarified, if this said something more like, "Creates a histogram of the values in a tensor"?

The thing that's a little weird for me in the current description is how should a user interpret the flattening, which is a shape property?